azure-iot-device/azure/iot/device/common/mqtt_transport.py

```diff
@@ -436,7 +437,7 @@ def connect(self, password=None):
             raise _create_error_from_rc_code(rc)
         self._mqtt_client.loop_start()
 
-    def disconnect(self, clear_pending=False):
+    def disconnect(self, clear_inflight=False):
```

Comment:
This is just a refactor for clarity. Honestly, this parameter will probably be entirely removed in the near-ish future, as this is too tightly coupled to disconnect for us to implement a robust cancellation flow.